runtests.py

```diff
@@ -2130,6 +2142,18 @@ def runtests(options, cmd_args, coverage=None):
                                 options.test_determinism,
                                 common_utility_dir, options.pythran_dir)
         test_suite.addTest(filetests.build_suite())
+    if options.examples and languages:
+        filetests = TestBuilder(options.examples_dir, WORKDIR, selectors, exclude_selectors,
+                                options.annotate_source, options.cleanup_workdir,
+                                options.cleanup_sharedlibs, options.cleanup_failures,
+                                options.pyregr,
+                                options.cython_only, languages, test_bugs,
+                                options.fork, options.language_level,
+                                options.test_determinism,
+                                common_utility_dir, options.pythran_dir,
+                                default_mode='compile',
+                                add_embedded_test=False)
```

Comment:
Hmm, I think seeing all these options makes it clear that this needs some kind of refactoring... Not sure which yet, but this doesn't look right...

Reply:
Passing the option variable? Then extract the members in the __init__()?